### PR TITLE
Add Settings view

### PR DESCRIPTION
**Pull request by vinaysshenoy**
https://www.pivotaltracker.com/story/show/166437675

The integration with the Mobius and the rest of the app will be done in the final PR after #851 gets merged.

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/settings/SettingsScreen.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package org.simple.clinic.settings
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.LinearLayout
+
+class SettingsScreen(
+    context: Context,
+    attributeSet: AttributeSet
+) : LinearLayout(context, attributeSet), SettingsUi {
```
Comment (rakshakhegde) [inline app/src/main/java/org/simple/clinic/settings/SettingsScreen.kt]: Multi-line comments!! 😃🌟

File: app/src/main/java/org/simple/clinic/settings/SettingsScreen.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+  override fun displayUserDetails(name: String, phoneNumber: String) {
+    TODO("not implemented")
```
Comment (vinaysshenoy) [inline app/src/main/java/org/simple/clinic/settings/SettingsScreen.kt]: This screen is not being used at the moment, so it's safe to leave this in.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -441,4 +441,11 @@
   <string name="simple_video_watch_video">Watch Video</string>
   <string name="simple_video_how_to_use_the_simple_app">How to use the Simple app</string>
   <string name="simple_video_duration">%s minutes</string>
+
+  <!-- Settings -->
+  <string name="settings_title">Settings</string>
+  <string name="settings_user_name_icon_content">Settings user name icon</string>
+  <string name="settings_user_phone_icon_content">Settings user phone number ccon</string>
```
Comment (ragunathjawahar) [inline app/src/main/res/values/strings.xml]: `ccon`, did you mean `icon`?

File: app/src/test/java/org/simple/clinic/settings/SettingsUiRendererTest.kt [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    // given
+    val name = "Anish Acharya"
+    val phoneNumber = "1234567890"
+    val modelWithUserDetails = defaultModel.userDetailsFetched(name, phoneNumber)
+
+    // when
+    renderer.render(modelWithUserDetails)
+
+    // then
+    verify(ui).displayUserDetails(name, phoneNumber)
```
Comment (ragunathjawahar) [inline app/src/test/java/org/simple/clinic/settings/SettingsUiRendererTest.kt]: Followed by,
`verifyNoMoreInteractions(ui)`?

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (vinaysshenoy) [reply app/src/main/res/values/strings.xml]: Whoops, typo! Will fix.

File: app/src/test/java/org/simple/clinic/settings/SettingsUiRendererTest.kt (hunk 4 above)
Comment (vinaysshenoy) [reply app/src/test/java/org/simple/clinic/settings/SettingsUiRendererTest.kt]: Good point, I'll add this.

**Comment by vinaysshenoy** [comment]
@ragunathjawahar Feedback addressed.

**Pull request merged**